utils/time.cc

```diff
@@ -1,4 +1,4 @@
-#include "cris/core/timer.h"
+#include "cris-core/time.h"
```

Comment:
Why changing cris/core to cris-core? cris/core seems fine to me, and it's cleaner.
And regardless of the reason, you'll need to update clang-format rules for enforce the ordering.